js/webapp.js

```diff
@@ -935,6 +935,9 @@ WebApp.prototype = {
             do404(req.url, res, this.options.productCode
             + ": unknown resource requested");
```

Comment:
This message should also be changed. It should say what was requested

Reply:
The message is not introduced by this pull request, it has been there for over a year, so please let me know what you want to be done. The only suggestion I have is
"Resource not found, URL: /ZLUX/plugins/com.rs.mvd/web/

Additional info: ZLUX: unknown resource requested"

Reply:
I just noticed these other occurrences of the same message when you added it to a new place. I feel that debugging for developers could be enhanced by at least putting more informative messages into the log, though, it if's possible to put more information into the 404 message without confusing an end-user, that would be nice, too. Even if the message to the end user was not directly useful to the user it might speed support calls if the error were able to be googled separately, and also, since the message would be in the UI it would be easier for an end-user to forward interesting information to whatever support is appropriate (without having to try to figure out which log message corresponded to their personal error)